satpy/scene.py

```diff
@@ -71,6 +71,13 @@ def __init__(self, filenames=None, reader=None, filter_parameters=None,
                  reader_kwargs=None):
         """Initialize Scene with Reader and Compositor objects.
 
+        Notice (see parameters list below) that it is possible to load a
+        combination of files or sets of files each requiring their specific
+        reader. For that ``filenames`` needs to be a `dict`, like e.g.::
+
+            scn = Scene(filenames={'nwcsaf-pps_nc': glob('/path/to/nwc/saf/pps/files/*'),
+                                   'modis_l1b': glob('/path/to/modis/lvl1/files/*')})
+
```

Comment:
This, being the less likely use case (especially by new users), should come after the below example(s). Preferably in the long run (would appreciate it in this PR, but I would understand if you don't have the time), this docstring would include a small example of creating a Scene with `reader` and `filenames` (a list), an empty Scene (as it does already), and with multiple readers (as you're adding here). I don't care about the order of the last two, but the reader+filenames case should definitely be the first thing users see.

We should also make sure that the generated sphinx API docs contain proper syntax highlighting for these code blocks. If they don't we should use `.. code-block:: python` instead of the generic `::`.

Reply:
Good suggestions, better now?

Reply:
Looks good. Could you try running the website generation locally and make sure these code fragments look good?

Reply:
Sure, I actually did that, and normally does that. But it would also be nice to have RTD built for PRs so that others can see. I think it looks okay. I was unsure to how much we should use the marking of what is a keyword. I kept only the first mentions of `reader` and `filenames` in italic, the following mentions are highlighted as we do for key word arguments.
![scene_reader_doc_example](https://user-images.githubusercontent.com/169069/161380054-27f6a903-77b7-4064-82ae-ea174868570a.png)